corehq/apps/locations/util.py

```diff
@@ -47,7 +47,7 @@ def loc_to_json(loc, project):
         SQLLocation.root_locations(
             domain, include_archive_ancestors=include_archived
         )
-        if user_can_view_location(user, loc, project)
+        if user is None or user_can_view_location(user, loc, project)
```

Comment:
This function is used in some places where we haven't added access restrictions, such as reports.  Maybe a more explicit way to do this would be to have two functions: `load_locs_json` and `load_locs_json_protected`.  Currently we're not limiting location access site-wide, just in the location edit pages.